README.md

```diff
@@ -18,38 +18,49 @@
 
 ### (Recommended) Using the pre-built image
 
-To be updated.
+Docker images are automatically published on when releases are tagged.
 
-### Other ways to build & run tsidp
+```bash
+# to use the latest image
+$ docker pull ghcr.io/tailscale/tsidp:latest
 
-<details>
-<summary>Building your own container</summary>
+# to use a specific release version
+$ docker pull ghcr.io/tailscale/tsidp:v0.0.2
+```
 
-Replace `YOUR_TAILSCALE_AUTHKEY` with your Tailscale authentication key in the following commands:
+Running a tsidp container:
 
-1. Use an existing auth key or create a new auth key in the [Tailscale dashboard](https://login.tailscale.com/admin/settings/keys). Ensure you select an existing [tag](https://tailscale.com/kb/1068/tags) or create a new one.
+> [!TIP]
+> Replace `YOUR_TAILSCALE_AUTHKEY` with your Tailscale authentication key in the following commands:
```

Comment:
`YOUR_TAILSCALE_AUTHKEY` doesn't appear bellow. It looks like the `TS_AUTHKEY` was removed.